decidim-core/app/views/decidim/messaging/conversations/_reply.html.erb

```diff
@@ -1,11 +1,11 @@
-<div class="add-message">
+<div class="conversation-reply conversation-reply--offset">
   <h5 class="section-heading">
     <%= t(".title") %>
   </h5>
 
   <%= form_for form, url: decidim.conversation_path(conversation.id), method: :put, remote: true do |f| %>
-    <%= f.text_area :body, label: false, rows: 4, required: true %>
+    <%= f.text_area :body, label: false, rows: 4, required: true,  placeholder: t(".placeholder") %>
```

Comment:
Hawk eye here, there's an extra space between `true,` and `placeholder`. I think it's not worth "fixing" it, but if you have another significant change to make, you could do it on the way :laughing:

Reply:
Solved!